docs/puzzle/templates/Laboratory/medical-disaster.md

```diff
@@ -0,0 +1,76 @@
+﻿# puzzle_template
+
+<!---
+    Bitte einen Kategorie-Ordner erstellen, falls noch nicht vorhanden.
+    /docs/puzzle/templates/*hier Kategorie Ordner einfügen*
+-->
+
+
+# Name
+
+<!---
+    -  Einen fancy Namen überlegen :)
+-->
+
+Freisetzung gefährlicher Chemikalien
+
+---
+
+# Beschreibung
+
+<!---
+    - Sollte das Rätsel nur mit den nötigsten Infos beschreiben.
+    - Dieser Abschnitt kann dem Spieler im HUD angezeigt werden.
+-->
+
+Stoppe die Freisetzung der Chemikalien.
+
+---
+
+# Ort
+<!---
+    - Wo ist dieses Rätsel zu finden? (in Wand integriert, freistehend,
+      über die Raumstation verteilt, ein ganzer Raum....)
+-->
+
+Rätsel ist in dem Labor der Raumstation zu finden.  
+
+---
+
+# Mechanik
+
+<!---
+    - Exakte Beschreibung der benötigten Schritte/Aufgaben des Spielers 
+-->
+
+Finden einer Stelle, bei der Chemikalien freigesetzt werden -> 
+Beheben des Problemes durch Interaktion mit dem Gerät (entweder einem Werkzeug oder
+das Drehen eines Ventiles.
+
+---
+
+# Endbedingungen
+
+<!---
+    - Exakte Beschreibung, wann das Rätsel erfolgreich gelöst ist.
+    - (optional) Exakte Beschreibung, wann es fehlschlägt.
+    - (optional) Exakte beschreibung, wann Rätsel zurückgesetzt wird.
+-->
+
+Rätsel erfolgreich gelöst, wenn der Spieler sämtliche Ventile geschlossen hat
+oder sonstige Interaktionen abgeschlossen sind.
```

Comment:
Kann das Rätsel fehlschlagen?
Bei Verlust z.B. eine variable Zeit vom Timer abziehen?
Oder eher dazu addieren, damit man länger Tasks machen muss?

Vllt müssen wir nochmal eine exakte Überlegung zur gesamten Gewinnbedingung anstellen.

Reply:
Hier haben wir ja ggf. in der Zukunft den Integritätswert, wenn der Task fehlschlägt, deshalb definiere ich
hier erstmal nur ein Fehlschlag für diesen Task

Reply:
Ja hast Recht, füge ich noch ein.